lib/active_model_serializers/rspec/matchers/schema.rb

```diff
@@ -0,0 +1,31 @@
+require 'active_model_serializers/test/schema'
+
+module ActiveModelSerializers
+  module RSpecMatchers
+    module Schema
+      extend ActiveSupport::Concern
+
+      included do
+        RSpec::Matchers.define :have_valid_schema do |schema_path|
```

Comment:
`Base` was just poor naming, now that I move my "fix" to `AssertSchema`, I can simply initialize that class and use it as intended.